.github/workflows/publish.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  workflow_call:
+    inputs:
+      file_path:
+        required: true
+        type: string
+      release_name:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      changelog:
+        required: true
+        type: string
+      cf_mc_version:
+        required: true
+        type: string
+      mr_mc_version:
+        required: true
+        type: string
+      mr_loader:
+        required: true
+        type: string
+    secrets: inherit
+
+jobs:
+  release-to-cf:
+    name: Release to CurseForge
+    runs-on: ubuntu-latest
+    needs: [changelog, artifact]
+    steps:
+      - uses: actions/download-artifact@v3
+
+      - name: Upload to CurseForge
+        uses: itsmeow/curseforge-upload@v3
+        with:
+          file_path: "${{ inputs.file_path }}"
+          game_endpoint: minecraft
+          game_versions: "${{ inputs.cf_mc_version }}"
+          project_id: 303451
```

Comment:
Should we not have our own project? This is the Wynntils project. I think it will be very confusing to users, publishing Artemis builds there. Or won't it?

Reply:
Personally I don't think so, we publish it as an alpha. And the sites clearly mark which MC version is targeted.